README.md

```diff
@@ -0,0 +1,134 @@
+| Username          | :heavy_check_mark: | Email address or username used authenticate on email server        |
+| Password          | :heavy_check_mark: | Password used authenticate on email server                         |
+| StartTLS          | false              | Enable StartTLS connection if server supports                      |
+| TLSAcceptAllCerts | false              | Accept insecure certificates with TLS connection                   |
+
+#### Mattermost
+
+Mattermost configuration
+
+| Field    | Required/Default   | Description                                                                                                                |
+|----------|:------------------:|----------------------------------------------------------------------------------------------------------------------------|
+| Server   | :heavy_check_mark: | Address of mattermost server. Please inform protocol and port if its necessary ex: _<https://mattermost.example.com:8065>_ |
+| Team     | :heavy_check_mark: | Team name. You can find teams name by [(URL)](https://github.com/stuvusIT/mattermost-mattermail#teamchannel)               |
+| User     | :heavy_check_mark: | User used to authenticate on Mattermos server                                                                              |
+| Password | :heavy_check_mark: | Password used to authenticate on Mattermos server                                                                          |
+| UseAPIv3 | true               | Set to use Mattermost Api V3                                                                                               |
+
+#### MailTemplate
+
+This configuration formats email message using markdown to post on Mattermost.
+The default configuration is `:incoming_envelope: _From: **{{.From}}**_\n>_{{.Subject}}_\n\n{{.Message}}`, in this example when Mattermail receives a message from `john@example.com`, with subject `Hello world` and message body `Hi I'm John`. This email will be formated to:
+
+:incoming_envelope: _From: **john@example.com**_
+>_Hello world_
+
+Hi I'm John
+
+#### RedirectBySubject
+
+If the option `RedirectBySubject` is `true` the Mattermail will try to redirect an email and post it using the subject, ex:
+
+| Subject                     | Destination                         |
+|-----------------------------|-------------------------------------|
+| [#orders] blah              | channel `orders`                    |
+| [#orders #info] blah        | channel `orders` and `info`         |
+| Fwd [#orders] [#info] blah  | channel `orders` and `info`         |
+| [1234#orders] foo           | channel `orders`                    |
+| [@john] blah                | user `john`                         |
+| [@john #orders] blah        | user `john` and channel `orders`    |
+
+#### Filter
+
+This option is used to redirect email following the rules.
+
+```javascript
+"Filter":            [
+    /* if subject contains 'Feature' redirect to #feature */
```

Comment:
Shouldn't this be yml?

Reply:
no yml layout of json is shit so we use javascript

Reply:
I mean the code itself should be yml. Or can I insert this json code in my host vars and it will work?